app/react/src/server/cra-config.js

```diff
@@ -62,7 +61,16 @@ export const getRules = extensions => rules =>
   }, []);
 
 const getStyleRules = getRules(cssExtensions.concat(cssModuleExtensions));
-const getTypeScriptRules = getRules(typeScriptExtensions);
+
+export const getTypeScriptRules = (webpackConfigRules, configDir) => {
+  const rules = getRules(typeScriptExtensions)(webpackConfigRules);
+  // Add support for using TypeScript in the `.storybook` (or config) folder.
+  const babelRule = rules[0];
+  // Resolves an issue where this config is parsed twice (#4903).
+  if (typeof babelRule.include !== 'string') return rules;
```

Comment:
I think the problem here is that you are mutating the initial object, instead of just returning a new instance

Reply:
Thanks @igor-dv, this is now resolved.